### ISSUE TITLE
Render archivy knowledge map

### ISSUE THREAD

**Issue by srevinsaju** [archivy/archivy#172]
A candy feature, interconnect different things on the knowledge base, helps us to visualize the second brain. It is not a productive feature, but it would surely attract a lot of users. Think of it like, 'cloudarchivy' similar to worldcloud,

something like this:

![image](https://user-images.githubusercontent.com/48695438/103466796-331e8b80-4d59-11eb-959b-e42934cf9711.png)

> ~ from [obsidian](https://obsidian.md)

**Comment by Uzay-G** [comment]
Definitely, we need to implement #168 and #119 first to have a working tag setup to generate that type of cloud.

**Comment by srevinsaju** [comment]
Cool!

**Comment by Uzay-G** [comment]
http://sigmajs.org/ could be useful - am looking into it.
